phase2/outline
# #236: Adding in mobile controller and CSS config changes to Outline Heading

**Developer Notes**
Adding in the `mobile controller` and CSS config changes into Outline Heading. Implements Issue #236  

**Testing Notes**
1. Go to Storybook
2. Go to the Heading component
3. Size the screen up and down
4. Observe that the text gets smaller at smaller screens and larger at larger screens

<a href="https://gitpod.io/#https://github.com/phase2/outline/pull/237"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
